h2o-genmodel/src/main/java/hex/genmodel/algos/deepwater/DeepwaterMojoModel.java

```diff
@@ -81,7 +81,7 @@
 
   static public BackendTrain createDeepWaterBackend(String backend) {
     try {
-//      if (backend.equals("caffe"))      return new DeepwaterCaffeBackend();
+      if (backend.equals("caffe"))      return new DeepwaterCaffeBackend();
```

Comment:
This should break all JUnits as it will think that the backend is available and then some tests will fail.